content/blog/announcing_kubeflow_0.3.md

```diff
@@ -0,0 +1,156 @@
++++
+title = "Kubeflow 0.3 Simplifies Setup & Improves ML Development"
+description = "Announcing Kubeflow 0.3"
+weight = 20
+publishDate = 2018-10-26T11:52:19-07:00
+draft = false
++++
+
+## Since Last We Met
+
+It has been 3 months since landing [0.2](https://www.kubeflow.org/blog/announcing_kubeflow_0.2/) and we have continued to be amazed by the support from the community. The community continues to grow, with 100+ members from 20 different organizations and more than 4,500 Github stars!
+
+In September, we held our first [contributor summit](http://bit.ly/kf-summit-2018-overview)!

```

Comment:
This can be merged with the next paragraphs.

Reply:
Done.